tests/lib/tools/snaps-state

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+show_help() {
+    echo "usage: make-snap [snap-name]"
+    echo "usage: install-local [snap-name]"
+    echo "usage: install-local-as [snap-name] [dest-name]"
+    echo "usage: install-local-devmode [snap-name]"
+    echo "usage: install-local-classic [snap-name]"
+    echo "usage: install-local-jailmode [snap-name]"
```

Comment:
Is this a multi-call script? Otherwise if those are subcommands it's usual to show the usage with $0 as the script invocation name (like "usage: $0 make-snap ...")

Reply:
@cmatsuoka thanks for the review, I already updated the help and most of those subcommands have disappear